src/sentry/static/sentry/app/components/assigneeSelector.jsx

```diff
@@ -62,7 +64,7 @@ const AssigneeSelector = React.createClass({
 
     return {
       assignedTo: group.assignedTo,
-      memberList: MemberListStore.getAll(),
+      memberList: MemberListStore.loaded ? MemberListStore.getAll() : null,
```

Comment:
what are your thoughts on using two variables for memeberlist,one to hold isloaded state, one to hold the data? versus having it be null. just curious, not saying it's def better

Reply:
I didn't want to expose `loaded` at all, I think I prefer that components can derive basic state based on data type (especially when data comes from async source). Initially I wanted the MemberListStore to initialize `items` as null instead of an empty array, but that would involve a larger change with the other components, so I opted for this instead. When you have a separate `loading` property, it's just another piece of state that you need to sync, i.e. on api response, instead of just updating data, you need to also update `isLoaded`.